katdal/datasources.py

```diff
@@ -102,6 +103,80 @@ def _apply_data_lost(orig_flags, lost, block_id):
     return flags
 
 
+def corrprod_to_autocorr(corrprod):
+    """Find the autocorrelation indices of correlation products.
+
+    Parameters
+    ----------
+    corrprod : sequence of 2-tuples or ndarray
```

Comment:
I would call this `corrprods` to match the rest of the code base.

Reply:
Just the parameter, or do you prefer to rename the function too?

Reply:
I only thought of the parameter, but good question... Probably both (you could also rename the function `autocorr_indices` for more options :-))